Form/FormMapper.php

```diff
@@ -200,9 +200,22 @@ public function create($name, $type = null, array $options = array())
     public function setHelps(array $helps = array())
     {
         foreach ($helps as $name => $help) {
-            if ($this->admin->hasFormFieldDescription($name)) {
-                $this->admin->getFormFieldDescription($name)->setHelp($help);
-            }
+            $this->addHelp($name, $help);
```

Comment:
this is not necessary, we could keep the old logic here, but IMO its better to rely on one logic